code/__DEFINES/mobs.dm

```diff
@@ -113,7 +113,6 @@
 #define CANROOT (1<<6)
 #define GODMODE (1<<12)
 #define FAKEDEATH (1<<13) //Replaces stuff like changeling.changeling_fakedeath
-#define DISFIGURED (1<<14) //I'll probably move this elsewhere if I ever get wround to writing a bitflag mob-damage system
```

Comment:
make it clear that 1<<14 is unused or move the rest down

Reply:
`(1<<14)` is now clearly marked as unused.